docs/src/reference/editions.md

```diff
@@ -0,0 +1,40 @@
+## Editions
+
+Editions of buffrs mark a specific evolutionary state of the package manager.
+The edition system exists to allow for fast development of buffrs while
+allowing you to already migrate existing protobufs to buffrs even though it
+has not yet reached a stable version.
+
+Editions can be either explicitly stated in the `Proto.toml` or are
+automatically inlined once a package is created using buffrs. This ensures that
+you dont need to care about them as a user but get the benefits.
+
+> Note: If you release a package with an edition that is incompatible with
+> another one (e.g. `0.7` is incompatible with `0.8`) you will need to
```

Comment:
I would write "(e.g. *if* `0.7` is incompatible with `0.8`)" here to highlight that `0.8` might be compatible with `0.7` but it's not guaranteed.